nats.go

```diff
@@ -884,6 +911,35 @@ func RootCAs(file ...string) Option {
 	}
 }
 
+// RootCAsMem is a helper option to provide the RootCAs pool from memory.
+// If Secure is not already set this will set it as well.
+func RootCAsMem(mem []byte) Option {
+
+	return func(o *Options) error {
+		rootCAsCB := func() (*x509.CertPool, error) {
+			pool := x509.NewCertPool()
+			for _, f := range mem {
```

Comment:
You're iterating over a byte slice here, and for each byte you're doing `ok := pool.AppendCertsFromPEM(mem)` - that does not make much sense. Probably the signature of this function should be:
```go
func RootCAsMem(rootCAs ...[]byte) Option {
```

And then you can iterate over the certificates.

Reply:
done.